### DISCUSSION TITLE
resetting the environment

### DISCUSSION THREAD

**Discussion by tomsilver** [empriselab/RCareWorld#97] [Help - Programming] [answered]
Is it possible to reset the environment without closing unity (which would be slow)? I'm looking for something like `env.reset` in the Gym API.

**Comment by YoruCathy** [comment, marked as answer]
We don't explicitly have a reset function for the environment, but with RCareWorld GymWrapper, you can use the reset() function from Gym to reset the Gym-related stuff. If you want to reset other objects in the environment, you need to write your own reset function.

**Comment by tomsilver** [comment]
Understood, thanks for the info!
